config/config_test.go

```diff
@@ -21,30 +79,139 @@ func TestReadConfig(t *testing.T) {
 				Version: "latest",
 				Image:   "screwdrivercd/launcher",
 			},
+			filePath: cnfPath,
 		}
 
-		cnfPath := path.Join(testDir, "successConfig")
-		actual, err := ReadConfig(cnfPath)
+		actual, err := New(cnfPath)
 
 		assert.Nil(t, err)
+		assert.Equal(t, actual.filePath, cnfPath)
```

Comment:
```suggestion
		assert.Equal(t, cnfPath, actual.filePath)
```
I think the order of the arguments is reversed.